src/bp/ui-admin/src/index.tsx

```diff
@@ -31,3 +32,5 @@ window['BlueprintJsSelect'] = BlueprintJsSelect
 
 initializeTranslations()
 ReactDOM.render(<div>{routes}</div>, document.getElementById('root'))
+
+startTelemetry()
```

Comment:
There should be an opt out mechanism and I don't see any. just like you send your telemetry url and set it to window, you should also send the `SEND_USAGE_STATS` variable.

if this is false then telemetry should not be started nor setup

Reply:
Fixed in : e135fcf